build/report.py

```diff
@@ -36,6 +36,25 @@ def getServer():
   return server
 #end
 
+def getCluster():
```

Comment:
i didn't even know how to do this

Reply:
There was no cool trick or API call or anything. I just had a look at the JSON returned by getServer, and changed the return value to get the cluster instead.